pkg/cmd/checkendpoints/controller/pod_network_connectivity_check_controller.go

```diff
@@ -67,6 +74,16 @@ func NewPodNetworkConnectivityCheckController(podName, podNamespace string,
 // Sync ensures that the status updaters for each PodNetworkConnectivityCheck is started
 // and then performs each check.
 func (c *controller) Sync(ctx context.Context, syncContext factory.SyncContext) error {
+	// if PNCC CRD is not installed and established, cleanup any existing checkers, block
+	established, err := c.podNetworkConnectivityCheckResourceDefinitionEstablished()
```

Comment:
make this "past the post".  Once you successfully see the CRD created, just assume it never goes away.  Separately, write a control loop that watches for the CRD to disappear and then exit(0)

Reply:
Updated.